packages/core/lib/commands/call/meta.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  command: "call",
+  description: "Call read-only contract function with arguments",
+  builder: {},
+  help: {
+    usage:
+      "truffle call <contract-name> <function-name> [<function-args...>] [--network=<network-name>",
+    usage:
+      "truffle call <contract-name> <function-name> [<args...>]\n" +
+      "                                " + // spacing to align with previous line
+      "[--network=<network-name>] [--from=<account>]",
+    options: [],
+    allowedGlobalOptions: ["from", "network", "config"]
```

Comment:
oh I just remembered... I also forgot about a blockNumber parameter (default `"latest"`)

Reply:
@haltman-at says "oh so you'll need CLI flags for every damn tx option"

Reply:
we can probably skip access lists tho